examples/node/index.js

```diff
@@ -1,8 +1,8 @@
-import { LiFi } from '@lifi/sdk'
+import { LiFi, ChainId, CoinKey, findDefaultToken } from '@lifi/sdk'
+import { RPCProvider } from '@lifi/rpc-wrapper'
+import { Wallet } from 'ethers'
 
-import ethers from 'ethers'
-
-const lifis = new LiFi()
+const lifi = new LiFi()
```

Comment:
We should also initialize LiFi in the JS example with the required integrator string to show the users it is always needed.

Reply:
Yes, I made the changes already but the push was pending 😄